### ISSUE TITLE
postinstall failing

### ISSUE THREAD

**Issue by wilsonianb** [coilhq/tigerbeetle-node#9]
Both running `yarn` in the repo and installing 0.3.0 as a dependency fails with yarn 2:
```
➤ YN0007: │ tigerbeetle-node@workspace:. must be built because it never did before or the last one failed
➤ YN0009: │ tigerbeetle-node@workspace:. couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-6dcc4af8/build.log)
➤ YN0009: │ tigerbeetle-node@workspace:. couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-6dcc4af8/build.log)
```
```
./src/tigerbeetle/src/state_machine.zig:269:34: error: no member named 'entry' in struct 'std.hash_map.GetOrPutResult'
            const exists = insert.entry.value;
                                 ^
./src/tigerbeetle/src/state_machine.zig:129:45: note: while checking if 'tigerbeetle.src.state_machine.StateMachine.commit' is async
            .create_accounts => self.execute(.create_accounts, input, output),
                                            ^
./src/tigerbeetle/src/vr/replica.zig:1474:72: note: while checking if 'tigerbeetle.src.vr.replica.Replica.commit_op' is async
        const reply_body_size = @intCast(u32, self.state_machine.commit(
                                                                       ^
./src/tigerbeetle/src/vr/replica.zig:1446:31: note: while checking if 'tigerbeetle.src.vr.replica.Replica.commit_ops_through' is async
                self.commit_op(prepare);
                              ^
./src/tigerbeetle/src/vr/replica.zig:1984:78: note: while checking if 'tigerbeetle.src.vr.replica.Replica.repair' is async
        if (self.commit_min < self.commit_max) return self.commit_ops_through(self.commit_max);
                                                                             ^
./src/tigerbeetle/src/vr/replica.zig:82:5: note: while checking this field
    appending_frame: @Frame(write_to_journal) = undefined,
    ^
./src/tigerbeetle/src/state_machine.zig:326:34: error: no member named 'entry' in struct 'std.hash_map.GetOrPutResult'
            const exists = insert.entry.value;
                                 ^
./src/tigerbeetle/src/state_machine.zig:414:19: error: no member named 'entry' in struct 'std.hash_map.GetOrPutResult'
            insert.entry.value = c;
                  ^
./src/tigerbeetle/src/state_machine.zig:447:26: error: no member named 'value' in struct 'std.hash_map.Entry'
            return &entry.value;
                         ^
./src/tigerbeetle/src/state_machine.zig:288:22: error: no member named 'removeAssertDiscard' in struct 'std.hash_map.HashMap(u128,tigerbeetle.src.tigerbeetle.Account,std.hash_map.AutoContext(u128),80)'
        self.accounts.removeAssertDiscard(a.id);
                     ^
./src/tigerbeetle/src/message_bus.zig:1080:60: error: no member named 'removeAssertDiscard' in struct 'std.hash_map.HashMapUnmanaged(u128,*tigerbeetle.src.message_bus.Connection,std.hash_map.AutoContext(u128),80)'
                            .replica => bus.process.clients.removeAssertDiscard(self.peer.client),
                                                           ^
./src/tigerbeetle/src/state_machine.zig:367:23: error: no member named 'removeAssertDiscard' in struct 'std.hash_map.HashMap(u128,tigerbeetle.src.tigerbeetle.Transfer,std.hash_map.AutoContext(u128),80)'
        self.transfers.removeAssertDiscard(t.id);
                      ^
./src/tigerbeetle/src/state_machine.zig:456:26: error: no member named 'value' in struct 'std.hash_map.Entry'
            return &entry.value;
                         ^
./src/tigerbeetle/src/state_machine.zig:465:26: error: no member named 'value' in struct 'std.hash_map.Entry'
            return &entry.value;
                         ^
./src/tigerbeetle/src/state_machine.zig:436:21: error: no member named 'removeAssertDiscard' in struct 'std.hash_map.HashMap(u128,tigerbeetle.src.tigerbeetle.Commit,std.hash_map.AutoContext(u128),80)'
        self.commits.removeAssertDiscard(c.id);
                    ^

```

**Comment by wilsonianb** [comment]
Duplicate of https://github.com/coilhq/tigerbeetle/issues/21#issue-913237281

**Comment by DonChangfoot** [comment]
thanks for catching this @wilsonianb.

closed by 99354fe81453df9e499db66ce21c2f7c9ffb5b64
